react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log(`Username: ${username}, Password: ${password}`);
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Username:
+        <input
+          type="text"
+          name="username"
+          onChange={(event) => setUsername(event.target.value)}
+        />
+      </label>
+
+      <label>
+        Password:
+        <input
+          type="password"
+          name="password"
+          onChange={(event) => setPassword(event.target.value)}
+        />
+      </label>
+
+      <button type="submit">Sign Up</button>
+    </form>
```

Comment:
@kevin-l1 Please make sure you're setting the input value props based upon the React app's state